java/developer-guide/presentation-slide/convert-slide/_index.md

```diff
@@ -154,15 +265,13 @@ try {
 
         // Convert slide to a Bitmap object
         BufferedImage bmp = pres.getSlides().get_Item(i).getThumbnail(2f, 2f);
-		
-        // Create file name for an image
-        String outputFilePath = outputDir + "Slide_" + i + ".jpg";
 
         // Save the image in PNG format
-        ImageIO.write(bmp, "PNG", new File(outputFilePath));
+        ImageIO.write(bmp, "PNG", new File("Slide_" + i + ".jpg"));
```

Comment:
Inconsistency in the output image format. Please choose one from JPEG or PNG.

Reply:
Done